contracts/interfaces/IAxelarGateway.sol

```diff
@@ -144,6 +144,12 @@ interface IAxelarGateway {
     |* Getters *|
     \***********/
 
+    // solhint-disable-next-line func-name-mixedcase
+    function AUTH_MODULE() external view returns (address);
```

Comment:
These function names are terrible. I'd rather keep those two vars internal/private and create getters for them ourselves instead of using the auto-generated ones.

Reply:
Yeah, I prefer that too

Reply:
Fixed